apps/super-app/src/routes/SupportDetailWip.tsx

```diff
@@ -1,17 +1,28 @@
 import { useEffect, useState } from 'react';
 import { Box, Typography, Link } from '@mui/material';
-import { Public as PublicIcon } from '@mui/icons-material';
+import {
+  ArrowBackIosNew as ArrowBackIosNewIcon,
+  Public as PublicIcon,
+} from '@mui/icons-material';
 import {
   Twc2SupportDetail,
   FacebookLinksSection,
+  SupportDetailStrings,
+  getSupportDetailStrings,
   getTwc2Detail,
 } from '../services';
 import { Container } from '../common/components';
 import { PrimaryButton } from '../common/components/RoundedButton';
 import './SupportDetailWip.css';
 
-function Header() {
-  return <nav className="support-header">Navigation</nav>;
+function Header(props: { title: string }) {
+  const { title } = props;
+  return (
+    <nav className="support-header">
+      <ArrowBackIosNewIcon />
```

Comment:
Can we add some spacing between the arrow and the header? It looks a little cramped right now.

Reply:
We'll tweak these in the next round of changes :)